runbotics/runbotics-orchestrator-ui/src/src-app/translations/en/components.json

```diff
@@ -6,6 +6,7 @@
 
     "Component.BotProcessRunner.Warning": "Looking for a bot",
     "Component.BotProcessRunner.Error": "All bots are busy",
+    "Component.BotProcessRunner.Error.GuestRunLimitExceeded": "Guests have limited runs. Register to get unlimited executions.",
```

Comment:
maybe better `You have reached your daily limit. Create an account to continue or come back tomorrow`

Reply:
done